assets/xml/objects/object_ds2n.xml

```diff
@@ -1,55 +1,84 @@
-        <Skeleton Name="object_ds2n_Skel_008170" Type="Normal" LimbType="Standard" LimbNone="OBJECT_DS2N_LIMB_NONE" LimbMax="OBJECT_DS2N_LIMB_MAX" EnumName="object_ds2n_Limbs" Offset="0x8170" />
-        <!-- <Blob Name="object_ds2n_Blob_008178" Size="0x8" Offset="0x8178" /> -->
+        <!-- Unused Potion Shop Seller from OoT, in that game (Ossan -> object_ds2) -->
+        <!-- [object_ds2] still exists as a copy from OoT in MM, but unused, this has a different skeleton with legs -->
+        <!-- Difficult to document, might have made mistakes, esp that second animation -->
+
+        <!-- Can't draw these in z64Utils, kinda obvious how the skeleton is mapped
+             But not sure which one is left/right. -->
+        <DList Name="gDs2nTorsoDL" Offset="0x2420" /> <!-- uses shirtbutton tex -->
+        <DList Name="gDs2nHeadDL" Offset="0x2740" /> <!-- uses mouth tex -->
+        <DList Name="gDs2n_DL_002C20" Offset="0x2C20" /> <!-- uses shirt tex -->
+        <DList Name="gDs2n_DL_002D38" Offset="0x2D38" /> <!-- uses shirt and skin tex -->
+        <DList Name="gDs2n_DL_002F68" Offset="0x2F68" /> <!-- uses fingers tex -->
+        <DList Name="gDs2n_DL_0030E0" Offset="0x30E0" /> <!-- uses shirt tex -->
+        <DList Name="gDs2n_DL_0031F8" Offset="0x31F8" /> <!-- uses shirt and skin tex -->
+        <DList Name="gDs2n_DL_0033B0" Offset="0x33B0" /> <!-- uses fingers tex -->
+        <DList Name="gDs2n_DL_003528" Offset="0x3528" /> <!-- uses pants tex -->
+        <DList Name="gDs2n_DL_0035F0" Offset="0x35F0" /> <!-- uses pants tex -->
+        <DList Name="gDs2n_DL_0036D0" Offset="0x36D0" /> <!-- uses pants tex -->
+        <DList Name="gDs2n_DL_0037A8" Offset="0x37A8" /> <!-- uses shoe and pants tex -->
+        <DList Name="gDs2n_DL_003988" Offset="0x3988" /> <!-- uses pants tex -->
+        <DList Name="gDs2n_DL_003A68" Offset="0x3A68" /> <!-- uses pants tex -->
+        <DList Name="gDs2n_DL_003B40" Offset="0x3B40" /> <!-- uses shoe and pants tex -->
+        
+        <Texture Name="gDs2nMainTlut" OutName="tlut_main" Format="rgba16" Width="16" Height="16" Offset="0x3CD8" />
+        <Texture Name="gDs2nFingersTlut" OutName="tlut_fingers" Format="rgba16" Width="16" Height="16" Offset="0x3ED8" />
+        <Texture Name="gDs2nEyesTlut" OutName="tlut_eyes" Format="rgba16" Width="16" Height="16" Offset="0x40D8" />
+        <Texture Name="gDs2nPantsAndShirtBottomTex" OutName="pants_and_shirt_bottom" Format="rgba16" Width="16" Height="16" Offset="0x42D8" />
+        <Texture Name="gDs2nShoesTex" OutName="shoes" Format="rgba16" Width="16" Height="32" Offset="0x44D8" />
+
+        <!-- from 48D8 -> 4CD8 is unacounted for, it is a dark green mix of pixels -->
+            <!-- <Blob Name="gDs2n_Blob_0048D8" Size="0xC00" Offset="0x48D8" /> -->
+        <!-- These two eye textures are corrupted, they extract and draw broken in-game.
```

Comment:
Hmm, there might be another explanation, but I'm not too worried about sorting it out since it's unused.